WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -243,6 +243,10 @@ private extension StorePickerViewController {
         preselectStoreIfPossible()
         reloadInterface()
     }
+
+    func backgroundColor() -> UIColor {
+        return WordPressAuthenticator.shared.unifiedStyle?.viewControllerBackgroundColor ?? .listBackground
```

Comment:
Isn't this style defined in our styles? 
I see a big optional change and also a singleton usage that makes me a bit nervous 😓

Reply:
It is, but... WordPresAuthenticator is a singleton, so there isn't much we can do there, and the style needs to match the style exposed by WPAuthenticator. 

That being said, in my opinion, exposing authentication as we do with any other global service would make sense in the long run.